client/components/Index.jsx

```diff
@@ -27,7 +27,7 @@ export default class Neighborhood extends React.Component {
 
   componentDidMount() {
     let queryString = window.location.search;
-    let listingId = (queryString.slice(-3) * 1)
+    let listingId = (queryString.slice(-8) * 1)
```

Comment:
We should set up a standard in writing with the team -- this will work if we treat every number as a string (i.e., listing number 250,087 is `id=00250087`) but not if we don't use leading zeros. I haven't made any changes to my client yet so either way is fine for me.

Reply:
Ok sounds good. I'll implement whatever we decide at the end.